package/MDAnalysis/analysis/base.py

```diff
@@ -243,39 +232,28 @@ def _compute(self, out_queue, order, progress):
             analysis_object._single_frame(timestep)
             progress.put(order) # Updates the progress bar
 
-        # Avoid initializing universe again when results are sent back
-        analysis_object._universe.filename = None
-        analysis_object._universe.trajectory.filename = None
-
-        out_queue.put((analysis_object, order)) # Returns the results
-
-    def _type(self):
-        return self.__class__.__name__
+        # Returns the results along with our order index
+        out_queue.put((analysis_object.results, order))
```

Comment:
Only put the results back into the out queue, who cares about the Analysis object at this point :)